curriculum/challenges/english/25-front-end-development/lecture-working-with-relative-and-absolute-units/672bb83c3a9906945536cff2.md

```diff
@@ -10,6 +10,53 @@ dashedName: what-are-vh-and-vw-units
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What are vh and vw units, and when should you use them?
+
+In CSS, vh and vw are viewport-relative units that allow you to size elements based on the dimensions of the browser window. These units are particularly useful for creating responsive designs that adapt to different screen sizes.
+
+vh stands for "viewport height," and 1vh is equal to 1% of the viewport's height.
+
+Similarly, vw stands for "viewport width," and 1vw is equal to 1% of the viewport's width. 
+
+This means that if you set an element's height to 100vh, it will occupy the full height of the viewport, regardless of the actual pixel dimensions of the device.
```

Comment:
```suggestion
This means that if you set an element's height to `100vh`, it will occupy the full height of the viewport, regardless of the actual pixel dimensions of the device.
```